packages/manager/modules/netapp/src/dashboard/index/controller.js

```diff
@@ -27,6 +30,24 @@ export default class OvhManagerNetAppDashboardIndexCtrl {
     );
   }
 
+  toggleNameEdition() {
+    this.isEditingName = !this.isEditingName;
+  }
+
+  editName() {
+    return this.$http
+      .put(`/storage/netapp/${this.storage.id}`, {
+        name: this.editNameValue,
+      })
+      .then((data) => {
+        console.log(data);
```

Comment:
remove console.log. Weird that you succeed to push with because commit -s should do a lint check